---
title: Mutations, gene expression and outcome
layout: post
---

**_How much do mutations screw up a cancer cell? And how is that related to outcome?_**

We have recently published a [paper in *Nature Communications*](http://dx.doi.org/10.1038/ncomms6901), where we analysed how mutations, gene expression, 
clinical phenotype and survival are connected in Myeolodysplasia, a form of blood cancer. In this study we took a panel of the 16 most frequently mutated genes 
and chromosomal aberrations in 124 patients with [Myeolodysplastic Syndromes (MDS)](http://www.cancerresearchuk.org/about-cancer/cancers-in-general/cancer-questions/myelodysplastic-syndromes-mds) and related those to
gene expression profiles and survival. 

What we found was that the set of **mutations correlate with vast gene expression changes affecting approximately 1/5 of the entire transcriptome**. That's interesting because
gene expression variability has been frequently observed, but it was not clear to which extent this inter-patient variability was a technical problem (noise) or true biology. Here we
could show that a large fraction - between 20% to 60% - can be explained by the presence (or absence) of mutations.

We next sought out to investigate how mutations and gene expression are related to survival. It is known that mutations and gene expression data can be used for prognosis, but they 
were never put head to head into a race. The surprise was that **gene expression data appear to win that competition, largely superseeding the effect of mutations**. 

*How can that be?* 

One hypothesis is that mutations misprogramme a cell as can be measured by a perturbed  gene expression profile. These perturbations then lead to 
aberrant blood counts and in turn to poor survival. Due to this hierarchical information flow it may be sufficient to know the state of the intermittent phenotypes such 
as gene expression data. 

Alternatively, there could be infrequent lesions for which we couldn't make any inference,  other germline variants and also non-hereditary or environmental influences on gene expression data and outcome.
So it may be interesting in future studies to investigate the effect of infrequent mutations using larger cohorts and also to assess the effect of germline variants with exomes or whole genome sequencing. 

*	M. Gerstung, A. Pellagatti, L. Malcovati, et al. (2015). [Combining gene mutation with gene expression data improves outcome prediction in myelodysplastic syndromes.]( http://dx.doi.org/10.1038/ncomms6901) *Nat Commun*, 6:5901.
